src/telemetry/telemetry.ts

```diff
@@ -30,20 +30,20 @@ class Telemetry implements ITelemetry {
   }
 
   public trackEvent(eventName: string, payload: any) {
-    this.appInsights.trackEvent({ name: eventName }, payload);
+    this.appInsights.trackEvent({ name: eventName, properties: payload });
```

Comment:
there should be a change here to re-enable telemetry so telemetry must not be flowing.